lib/utils.js

```diff
@@ -107,7 +107,11 @@ function patchEventTargetMethods(obj) {
 
     handler._fn = fn;
     handler._bound = handler._bound || {};
-    arguments[1] = handler._bound[eventName] = zone.bind(fn);
+
+    var listenerZone = global.zone;
+    arguments[1] = handler._bound[eventName] = function() {
```

Comment:
Any reason you can't just do `arguments[1] = handler._bound[eventName] = zone.bindOnce(fn);` here?